src/test/java/mate/academy/boot/bootdemo/service/ProductServiceTest.java

```diff
@@ -0,0 +1,51 @@
+package mate.academy.boot.bootdemo.service;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.boot.bootdemo.model.Product;
+import mate.academy.boot.bootdemo.repository.ProductRepository;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+public class ProductServiceTest {
+    private AnnotationConfigApplicationContext context
+            = new AnnotationConfigApplicationContext("mate.academy.boot.bootdemo");
+    private final ProductService productService = context.getBean(ProductService.class);
```

Comment:
If you want to test your service - use Mockito. Try to avoid autowiring the real beans